platform/ext/target/nordic_nrf/nrf5340/target_cfg.c

```diff
@@ -193,3 +193,59 @@ int32_t spu_init_cfg(void)
     return ARM_DRIVER_OK;
 }
 
+int32_t spu_periph_init_cfg(void)
+{
+	/*
+	 * Grant Non-Secure access to nRF peripherals that are not Secure-only
```

Comment:
I suggest moving the comment to the header file.

Reply:
Yeap, I can do that.

Reply:
fixed now